src/store/getters/state-getters.ts

```diff
@@ -57,7 +57,7 @@ export const getFilingName = (state: StateIF): FilingNames => {
     case FilingTypes.INCORPORATION_APPLICATION: return FilingNames.INCORPORATION_APPLICATION
     case FilingTypes.REGISTRATION: return FilingNames.REGISTRATION
     case FilingTypes.RESTORATION: return FilingNames.RESTORATION_APPLICATION
-    case FilingTypes.VOLUNTARY_DISSOLUTION:
+    case FilingTypes.DISSOLUTION:
       return isTypeFirm(state) ? FilingNames.DISSOLUTION_FIRM : FilingNames.VOLUNTARY_DISSOLUTION
```

Comment:
As you see here, this UI assumes that a dissolution filing is a voluntary dissolution. That works today and we can fix this up in the future if needed.

(We might have to check the dissolution filing subtype if we need to know what kind of dissolution it is. Today, this UI only handles voluntary dissolutions.)